### DISCUSSION TITLE
Mesh duplication

### DISCUSSION THREAD

**Discussion by FrankFrank9** [pyvista/pyvista#6084] [Q&A]
### Describe the feature you would like to be added.

Hello,

I was wondering if there is the possibility of adding a mesh duplication only at the phase when screenshot is saved to save up memory. 
For reference: say I have a mesh with a sphere and all I need is for visualisation purposes to duplicate that sphere within the same image (say 1 more sphere up and 1 more sphere down). At the moment I achieve that with `translate("translation direction", inplace=False)` which copies all the mesh and solution arrays as well requiring a huge extra amount of memory if the mesh is big. Is there any other "memory friendly" option? If not would it be possible to do just a graphical duplication?

**Comment by MatthewFlamm** [comment]
I don't think this solves the memory problem, but I find using `glyph` makes generating many identical objects faster.
